src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/scheduler/RCAScheduler.java

```diff
@@ -72,6 +65,7 @@
   ThresholdMain thresholdMain;
   Persistable persistable;
   static final int PERIODICITY_SECONDS = 1;
+  static final int PERIODICITY_IN_MS = PERIODICITY_SECONDS * 1000;
   ScheduledFuture<?> futureHandle;
```

Comment:
I see that the users of the field are deleted. we should remove this too ? Did I miss something ?